generator/src/main/java/org/corfudb/generator/operations/OptimisticTxOperation.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.generator.operations;
+
+import java.util.List;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.generator.State;
+
+/**
+ * Created by maithem on 7/14/17.
+ */
+@Slf4j
+public class OptimisticTxOperation extends Operation {
+
+    public OptimisticTxOperation(State state) {
```

Comment:
it would be nice to add a tunable mixing ratio (read/write/transactional operations)